examples.py

```diff
@@ -0,0 +1,77 @@
+from gi.repository import Gio, GLib
```

Comment:
I think pygobject requires you to specify a version for gi.repository modules nowadays, or you get a warning. `gi.require_version('GLib', '2.0')` (check this in other files as well)